### PR TITLE
Create a EKS Cluster That Supports ADOT Operator Tests

### PR DESCRIPTION
**Pull request by sethAmazon**

### CODE REVIEW

File: terraform/eks_adot_operator_cluster_setup/main.tf [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        }
+      },
+    ]
+  })
+
+  managed_policy_arns = [
+    "arn:aws:iam::aws:policy/AmazonPrometheusRemoteWriteAccess",
+    "arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess",
+    "arn:aws:iam::aws:policy/CloudWatchAgentAdminPolicy",
+    "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess",
```
Comment (aateeqi) [inline terraform/eks_adot_operator_cluster_setup/main.tf:103]: what will we need readonly s3 access for?

Comment (sethAmazon) [reply terraform/eks_adot_operator_cluster_setup/main.tf:103]: Our sample app reaches out to s3 for extra traces

**Review by aateeqi — APPROVED** [review]

**Review by khanhntd — APPROVED** [review]
LGTM with "stop gap" strategy: tear down when fail and continue to run when success.

**Review by Aneurysm9 — APPROVED** [review]

**Comment by khanhntd** [comment]
Anyway for us to create this in a separate folder though? Maybe group it with ec2_setup and setup ?

**Pull request merged**